docs/components/checkbox-field/index.md

```diff
@@ -204,7 +204,7 @@ Target the error block via `data-error`.
 <div class='mb-4 w-64'>
   <Form::Fields::Checkbox
   >
-    <:label>Label <svg class="inline" xmlns="http://www.w3.org/2000/svg" width="24" height="24" stroke="currentColor" viewBox="0 0 24 24"><path d="M12 3a9 9 0 11-6.364 2.636A8.972 8.972 0 0112 3zm0 4.7v5.2m0 3.39v.01" fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></path></svg></:label>
+    <:label>Label <svg class="inline w-4 h-4 -mt-1" xmlns="http://www.w3.org/2000/svg" width="24" height="24" stroke="currentColor" viewBox="0 0 24 24"><path d="M12 3a9 9 0 11-6.364 2.636A8.972 8.972 0 0112 3zm0 4.7v5.2m0 3.39v.01" fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></path></svg></:label>
```

Comment:
I mentioned this in the PR description, but this is an update to our examples with the `:label` named block so that the icon looks not as big and bulky.

<img width="400" alt="Screenshot 2023-06-20 at 7 56 39 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/311edbba-deeb-4fd5-afd6-6ec35bab9bd1">